### PR TITLE
Add rbd v0.1.1

### PR DESCRIPTION
**Pull request by niusmallnan**
https://github.com/rancher/rancher/issues/8722#issuecomment-345338650

### CODE REVIEW

**Comment by deniseschannon** [comment]
@niusmallnan @yasker do we want to keep the old version available to users or have users use the latest available?

If we want to limit the old one, we should put a max rancher version on the previous template. Also, we should update this template to the release that it will be tested with (v1.6.13-rc1).

**Comment by yasker** [comment]
@deniseschannon 

@niusmallnan has updated PR according to your comment.

File: infra-templates/rbd/0/rancher-compose.yml [hunk 1]
```diff
@@ -4,6 +4,7 @@
   description: |
     Docker volume plugin for Ceph RBD
   minimum_rancher_version: v1.6.8-rc1
+  maximum_rancher_version: v1.6.8-rc1
```
Comment (deniseschannon) [inline infra-templates/rbd/0/rancher-compose.yml]: this needs to be at v1.6.12, otherwise there is no option for the releases

Comment (niusmallnan) [reply infra-templates/rbd/0/rancher-compose.yml]: @deniseschannon Done.

**Pull request merged**